.travis.yml

```diff
@@ -3,5 +3,17 @@ sudo: false
 go:
   - 1.5
   - 1.6
+  - 1.7
+
+before_install:
+  - go get github.com/mattn/goveralls
```

Comment:
It does not affect the tests when you run them locally. It's only used by travis.